### DISCUSSION TITLE
Question: dance() for custom gf animations not working

### DISCUSSION THREAD

**Discussion by GamingInfinite** [KadeArchive/Kade-Engine#765] [Q&A]
Context:
This is my first time trying to modify anything related to FNF coding wise so bare with me.

With the new release of the kade engine, custom keybinds are a thing now.  And I know that Whitty mod as well as other mods based on the kade engine will be ported probably at some point.  But as a first entry into trying to mod stuff I was trying to port Whitty to KE 1.5.1.  Although, whenever I tried to boot anything Whitty-related (eg. trying to play the story mode week or opening any song in free play) it would crash.  All the songs showed properly as well as weeks, but on hitting enter and going into gameplay it crashed.  Using -debug it's saying that !animation.curAnim.name.startswith("hair") is referencing a null object.  But every other song is working fine, and I'm 99% sure I setup everything else correctly.

**Comment by puyoxyz** [comment]
change `"hair` to `"hair"`
